enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -862,24 +862,35 @@ public QueryDeparturesResult queryDepartures(final String stationId, final @Null
 			// to stop_point and query departures.
 			final StringBuilder queryUri = new StringBuilder();
 			queryUri.append(uri());
-			if (equivs)
+			final String header = stationId.substring(0, stationId.indexOf(":"));
```

Comment:
Hmmm, looks like not all Navitia IDs are separated by ":". For example the IDs used in the live tests of the OntarioProvider.